test/bigtest/tests/mapping-profiles/mapping-profile-details-test.js

```diff
@@ -116,8 +121,333 @@ const hasTable = (details, accordion, table, tableName, columnCount, columnHeade
   });
 };
 
+const hasBooleanActionsField = (details, accordion, field, fieldLabel, selectedOption, optionToSelect, isMCL = false) => {
+  describe('When boolean actions decorator applies', () => {
+    it(`has correct ${fieldLabel} field label`, () => {
+      expect(mappingProfileForm[details][accordion][field].label).to.equal(fieldLabel);
+    });
+
+    it(`has ${fieldLabel} select`, () => {
+      expect(mappingProfileForm[details][accordion][field].hasSelect).to.be.true;
+    });
+
+    it('has selected default option', () => {
+      // @TODO: selectedOption argument should be re-checked and possibly removed after fix issue with initial data set up (UIDATIMP-500)
+      expect(mappingProfileForm[details][accordion][field].val).to.equal(selectedOption);
+    });
+
+    describe(`when option "${optionToSelect}" selected and form is submitted`, () => {
+      beforeEach(async () => {
+        await mappingProfileForm[details][accordion][field].selectAndBlur(optionToSelect);
+        await mappingProfileForm[details][accordion][field].blurSelect();
```

Comment:
Are you sure you need this action? Above you do select and **blur**

Reply:
Removed extra action